statsmodels/imputation/tests/test_ros.py

```diff
@@ -1,7 +1,5 @@
 from __future__ import division
 
-__test__ = False
```

Comment:
keep this in here conditional on pandas < 0.14, so we just skip unit tests for older pandas

pandas had limited assert support in <0.14. IIRC, I had problems with older pandas versions at some point and just avoided almost all the pandas testing asserts